dev/com.ibm.ws.microprofile.faulttolerance.cdi/resources/com/ibm/ws/microprofile/faulttolerance/cdi/resources/FaultToleranceCDI.nlsprops

```diff
@@ -124,6 +124,11 @@ retry.parameter.invalid.value.CWMFT5019W=CWMFT5019W: The Retry policy jitter del
 retry.parameter.invalid.value.CWMFT5019W.explanation=The jitter delay must be less than the delay. The jitter delay consists of the jitter parameter and the jitterDelayUnit parameter. The delay consists of the delay parameter and the delayUnit parameter.
 retry.parameter.invalid.value.CWMFT5019W.useraction=Correct the value of one or more of the parameters so that the duration for the jitter delay is less than the duration of the delay.
 
+# A method does not return Future or CompletionStage
+asynchronous.method.not.returning.future.completionstage.CWMFT5020E=CWMFT5020E: The asynchronous method {0} does not have a return type of either Future or CompletionStage.
```

Comment:
Change "The asynchronous method {0} does not have a return type of either Future or CompletionStage." to "The {0} asynchronous method does not have a return type of either java.util.concurrent.Future or java.util.concurrent.CompletionStage.".

Reply:
Updated, thanks